packages/block-editor/src/components/duotone-control/style.scss

```diff
@@ -6,11 +6,13 @@
 
 	.components-circular-option-picker {
 		padding: $grid-unit-15;
+		box-sizing: border-box;
```

Comment:
Is this happening for anything in a popover?

Maybe we can add border-box styling to all descendants of this popover wrapper element, rather than modifying individual components:
https://github.com/WordPress/gutenberg/blob/trunk/packages/customize-widgets/src/components/customize-widgets/index.js#L60

Reply:
I tried to make the fix more general though so that it will work out of the box across editors, or even when the component is used independently outside of any editor.

If we expect the style to be defined when using these components, then we should probably document it somewhere somehow.

Reply:
I think it is expected to be defined globally, otherwise wouldn't ever single element need to have this style rule?

Agreed that it could probably be documented.

Reply:
> otherwise wouldn't ever single element need to have this style rule?

That's my expectation though 🤔.

Reply:
On searching, I do see the style rule defined a lot more than I initially thought I would.

It looks like most places are using this reset mixin to apply `border-box`:
https://github.com/WordPress/gutenberg/blob/trunk/packages/edit-widgets/src/style.scss#L27

Heres' the definition:
https://github.com/WordPress/gutenberg/blob/2db0d5c100c6560915e53384beb2f51ed7180c34/packages/base-styles/_mixins.scss#L392-L400

Seems a bit inconsistent.

Reply:
the post editor does this 😓 
https://github.com/WordPress/gutenberg/blob/2db0d5c100c6560915e53384beb2f51ed7180c34/packages/edit-post/src/style.scss#L62-L72

Reply:
I included the reset CSS in 9d08356.

But c.c. @youknowriad, what do you think should be preferable way to do this? Include the reset CSS globally, or inline the necessary resets to the component. What's the convention now?